### DISCUSSION TITLE
Installing to CentOS - can't find an x86_64 download

### DISCUSSION THREAD

**Discussion by CommanderTso** [FiloSottile/age#442] [Q&A]
Hey folks,

I'm trying to install age to a Centos 7 machine.  I'd like to avoid a third party repository, but trying to download via filippo.io isn't working.  The URL I'm attempting is: `https://dl.filippo.io/age/latest?for=linux/x86_64`.  

Can anyone point me in the right direction on this? Thanks.

**Comment by FiloSottile** [comment]
Hi! That link seems to work for me, can you elaborate on what’s happening?

**Reply by CommanderTso** [reply]
Weird.  When I hit that link from any of my browsers (Chrome, MacOS, and Safari), I get a GitHub 404 page stating "Not Found". Same with curl:

```➜  curl -k -XGET "https://dl.filippo.io/age/latest?for=linux/x86_64"
<a href="https://github.com/FiloSottile/age/releases/download/v1.0.0/age-v1.0.0-linux-x86_64.tar.gz">Moved Permanently</a>.

➜  curl -k -XGET "https://github.com/FiloSottile/age/releases/download/v1.0.0/age-v1.0.0-linux-x86_64.tar.gz"
Not Found%```

Looking at the releases page, I also don't see any x86 version under either the new release or under v1.0.0.

Thanks very much for any help you can lend here.

**Reply by FiloSottile** [reply]
Huh, not sure why it loaded on my phone, but that's not the right link. x86_64 is amd64 in Go's nomenclature, which is what we use here, so you want https://dl.filippo.io/age/latest?for=linux/amd64

**Comment by CommanderTso** [comment]
Ah! My bad - thanks very much.
